packages/redis/src/lib/RedisProvider.ts

```diff
@@ -21,27 +18,29 @@ import {
   JoshProvider,
   MathOperator,
   Method,
-  Payloads,
-  PROPERTY_NOT_FOUND,
-  setProperty
-} from '@joshdb/core';
-import { Serialize } from '@joshdb/serialize';
+  Payloads
+} from '@joshdb/provider';
+import { SerializeJSON, toJSON, toRaw } from '@joshdb/serialize';
 import { isNullOrUndefined, isNumber, isPrimitive } from '@sapphire/utilities';
+import { deleteProperty, getProperty, hasProperty, PROPERTY_NOT_FOUND, setProperty } from 'property-helpers';
 import { createClient, RedisClientOptions, RedisClientType } from 'redis';
 import { v4 } from 'uuid';
 
 export class RedisProvider<StoredValue = unknown> extends JoshProvider<StoredValue> {
   public declare options: RedisProvider.Options;
+  public version: JoshProvider.Semver = { major: 2, minor: 0, patch: 0 };
```

Comment:
- Should be changed to a getter using version injector.
- Spacing between property declarations.